# Pan-tilt camera
KEY_POS_PAN : 'u'
KEY_NEG_PAN : 'i'
KEY_POS_TILT : 'o'
KEY_NEG_TILT : 'p'

# Misc/high level actions
KEY_NOOP : ''
KEY_RESET : 'r'

# Base control

# Arm control
DELTA_ANGLE_CAMERA : 0.08
DELTA_POSITION : 0.02
START_TILT : 0.85
START_PAN : 0

ROSTOPIC_JOINT_STATES : '/joint_states'
ROSTOPIC_SET_PAN : '/pan/command'
ROSTOPIC_SET_TILT : '/tilt/command'
ROSTOPIC_JOINT_COMMAND : '/joint_command'
ROSTOPIC_SET_JOINT : '/goal_dynamixel_position'
ROSTOPIC_BASE_COMMAND : '/base/cmd_vel'
ROSTOPIC_TELEOP : '/cmd_teleop'
ROSTOPIC_STOP_EXECUTION : '/stop_execution'
ROSTOPIC_HARDWARE_STATUS : '/arm/status'
ROSTOPIC_SAFETY_STATUS : '/teleop/safety_status'

WAIT_TIME : 0.5
